rust/core-lib/src/config.rs

```diff
@@ -199,13 +259,14 @@ impl ConfigManager {
         let mut settings: Config = settings.try_into().unwrap();
         // relative entries in plugin search path should be relative to
         // the config directory.
+        if let Some(ref config_dir) = self.config_dir {
```

Comment:
I think this needs an indent.

Reply:
👍. For Fuchsia I suspect we will opt out of file-based config stuff altogether, and just use the (yet to be implemented) RPC protocol methods for managing configs.